### PR TITLE
[prosemirror-view] Optional `side` param for `coordsAtPos`

### PR DESCRIPTION
**Pull request by aried3r**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  https://github.com/ProseMirror/prosemirror-view/blob/master/CHANGELOG.md#new-features
  https://github.com/ProseMirror/prosemirror-view/commit/250d4337216b5c24ccd4a847cc8e29c13694a33b#diff-1fdf421c05c1140f6d71444ea2b27638R278-R286
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/prosemirror-view/index.d.ts [hunk 1]
```diff
@@ -323,7 +323,7 @@ export class EditorView<S extends Schema = any> {
    * and `right` will be the same number, as this returns a flat
    * cursor-ish rectangle.
    */
```
Comment (mmorearty) [inline types/prosemirror-view/index.d.ts:325]: ^ Can you please copy over the new comment from https://github.com/ProseMirror/prosemirror-view/blob/13b046a834b489530a98dd362fa55703e52e076d/src/index.js#L279-L284 ?

**Review by mmorearty — CHANGES_REQUESTED** [review]

File: types/prosemirror-view/index.d.ts (hunk 1 above)
Comment (aried3r) [reply types/prosemirror-view/index.d.ts:328]: Good catch. Done!

**Review by mmorearty — APPROVED** [review]
looks good, thanks @aried3r!

**Comment by aried3r** [comment]
Ready to merge

**Pull request merged**